src/main/java/com/sap/broker/budgie/configuration/behavior/BehaviorConfiguration.java

```diff
@@ -0,0 +1,44 @@
+package com.sap.broker.budgie.configuration.behavior;
+
+import java.util.List;
+import java.util.UUID;
+
+public class BehaviorConfiguration {
+
+    private UUID id;
```

Comment:
What is this used for?

Reply:
Could you introduce better naming? and also for the class?

Reply:
It is not needed anymore, I will remove it.